### PR TITLE
Updated emailRecipients to allow zero to recipients.

### PR DESCRIPTION
**Pull request by anmolbohra97**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by AliRafiee — COMMENTED** [review]
Please take a look at the live tests and maybe add a new test that only adds cc and bcc without to.

File: sdk/communication/Azure.Communication.Email/src/Models/EmailRecipients.cs [hunk 1]
```diff
@@ -22,10 +22,10 @@ public partial class EmailRecipients
         /// <param name="bcc">Email bcc recipients. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="to"/> is null. </exception>
         public EmailRecipients(
```
Comment (AliRafiee) [inline sdk/communication/Azure.Communication.Email/src/Models/EmailRecipients.cs:24]: Feels like we should remove the constructor for EmailRecipients in the generated file.
I'm not a 100% sure how to do that without changing the SWAGGER.

Are we not going to change the SWAGGER to not make To not required?

Comment (anmolbohra97) [reply sdk/communication/Azure.Communication.Email/src/Models/EmailRecipients.cs:24]: Yeah, we are going to update the swagger. Komivi mentioned that there are other changes coming too so plan is to update all of them together. This change is a temporary workaround to unblock customers.

**Review by kagbakpem — APPROVED** [review]

**Comment by pallavit** [comment]
@joseharriaga could you please help them through the review.

**Pull request merged**